### PR TITLE
fix: bump tikisdk 3.0.0

### PR DESCRIPTION
**Pull request by ricardobrg**
Fix iOS sdk initialization

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -22,6 +22,7 @@ vue3-example-android: vue3-example
 	cd example/vue3 && npx cap run android
 
 vue3-example-ios: vue3-example
+	cd example/vue3/ios/App pod deintegrate
```
Comment (timoguin) [inline makefile:25]: Missing an `&&` here, I think.

File: makefile [hunk 2]
```diff
@@ -41,6 +42,7 @@ vue2-example-android: vue2-example
 	cd example/vue2 && npx cap run android
 
 vue2-example-ios: vue2-example
+	cd example/vue2/ios/App pod deintegrate
```
Comment (timoguin) [inline makefile:45]: Also missing an `&&` here.

**Review by timoguin — CHANGES_REQUESTED** [review]

File: makefile (hunk 1 above)
Comment (ricardobrg) [reply makefile]: fixed... in line 26 it was missing too

**Review by timoguin — APPROVED** [review]
👍

**Pull request merged**